src/omero_cli_render.py

```diff
@@ -257,14 +258,16 @@ def _configure(self, parser):
 
         render_type = ProxyStringType("Image")
         src_help = ("Rendering def source in the form <object>:<id>. "
-                    "Image is assumed if <object>: is omitted.")
+                    "Image is assumed if <object>: is omitted. Object "
+                    "can be Image, Project, Dataset, Plate or Screen")
```

Comment:
This renders as a very long line. 
![screen shot 2018-05-04 at 14 12 34](https://user-images.githubusercontent.com/2478303/39629810-4f14e442-4fa5-11e8-871e-26bb4cca4f65.png)

Maybe we should rather move the sentence ``Object can be...`` to a place behind ``Set rendering settings..`` just like it is in the ``copy`` subcommand for targets. There should be enough space for that.

This pays for

   - test
   - info
   - copy
   - set